src/vitest/describe/scenarioOutline.ts

```diff
@@ -42,7 +42,7 @@ export function describeScenarioOutline (
                 .checkIfStepExists(stepType, stepDetails)
 
             scenarioStepsToRun.push({
-                key : `${stepType} ${stepDetails}`,
+                key : `${stepType}: ${stepDetails}`,
```

Comment:
Following the standard Cucumber syntax, it makes sense to leave the colon out of the step key.

> ["When Cucumber tries to execute a step, it looks for a matching step definition to execute."](https://cucumber.io/docs/gherkin/reference/#:~:text=when%20cucumber%20tries%20to%20execute%20a%20step%2C%20it%20looks%20for%20a%20matching%20step%20definition%20to%20execute.)

```suggestion
                key : `${stepType} ${stepDetails}`,
```

Reply:
Yes, I wanted to add colon to Scenario etc.
I will remove it for steps ;).